pkgs/tools/security/rnp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, asciidoctor
+, botan2
+, bzip2
+, cmake
+, fetchFromGitHub
+, gnupg
+, gtest
+, json_c
+, pkg-config
+, python3
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rnp";
+  version = "0.15.1";
+
+  src = fetchFromGitHub {
+    owner = "rnpgp";
+    repo = "rnp";
+    rev = "v${version}";
+    sha256 = "1l7y99rcss5w24lil6nqwr0dzh2jvq0qxmdvq7j5yx3fdssd5xsv";
+  };
+
+  buildInputs = [ zlib bzip2 json_c botan2 gtest python3 gnupg ];
```

Comment:
check only inputs should be moved to checkInputs.

Reply:
Thanks for the suggestion.  I tried moving `gtest` to `checkInputs` but `buildPhase` would fail.  Since it is a build-time dependency, I've decided to move it to `nativeBuildInputs` instead.